src/desktop/package.json

```diff
@@ -122,12 +122,13 @@
     "@babel/preset-typescript": "^7.10.4",
     "@babel/register": "^7.10.5",
     "@teamsupercell/typings-for-css-modules-loader": "^2.2.1",
+    "@webpack-cli/serve": "^1.5.2",
     "babel-plugin-dev-expression": "^0.2.2",
     "chalk": "^4.1.0",
     "concurrently": "^5.3.0",
     "cross-env": "^7.0.2",
     "detect-port": "^1.3.0",
-    "electron": "^8",
+    "electron": "^14.0.0",
```

Comment:
`src/desktop/package-lock.json` could be removed as we use `yarn` mostly, there is yarn lock file already. Also did you test with electron 14 all functionality? This is major version bump, need thorough testing.